### PR TITLE
Elevator doof

### PR DESCRIPTION
**Pull request by ManiacMoustache**
Amalgam mess of pneumatics and elevator. Both work but this is a bad pr

### CODE REVIEW

File: components/pneumatics.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ def setup(self):
         Setup to enable everything after variable injection from robot.py. This is where the bulk of setup for this class should be.
         on_enable() may need to be used for when something needs to happen everytime the state is changed, like from autonomous to teleop.
         """
-        self.solenoid = wpilib.DoubleSolenoid(0, 1) #I don't know if this is the proper a) class to define or b) the right channels for the solenoid. This is mostly a placeholder and can be fixed
+        self.solenoid = wpilib.DoubleSolenoid(6, 7) #I don't know if this is the proper a) class to define or b) the right channels for the solenoid. This is mostly a placeholder and can be fixed
```
Comment (ChrisHirsch) [inline components/pneumatics.py]: Fixed the comment (remove) now that we know the right values

File: components/elevator.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+class Elevator:
+    motorsList: dict
+
+    def on_enable(self):
+        self.upSpeed = 0
+        self.motors = self.motorsList
```
Comment (ChrisHirsch) [inline components/elevator.py]: lets make this simpler..ie just use self.motorsList

File: components/elevator.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        self.upSpeed = 0
+        self.elevatorMotor = self.motorsList['elevatorMotor']
+
+        print("Elevator Enabled")
+    
+    def setSpeed(self, speed):
+        self.upSpeed = speed
+
+    def setRaise(self):
+        self.upSpeed = .7
```
Comment (tuffinmuffin) [inline components/elevator.py]: Make these tunables so we can adjust them later if needed.

Comment (ManiacMoustache) [reply components/elevator.py]: did that

**Pull request merged**